### DISCUSSION TITLE
SwaggerUI specify default values

### DISCUSSION THREAD

**Discussion by yevon** [PostgREST/postgrest#2868] [Q&A] [answered]
Is there any way to provide default values for swaggerUI so when you use the endopint, it has some default data so you just can call the endpoint with all filled defaults? Something like this:

![image](https://github.com/PostgREST/postgrest/assets/121157/2cc09b13-e7d6-4df7-a318-f83110528c87)

Thanks!

**Comment by laurenceisla** [comment, marked as answer]
That is not implemented yet. For now, a solution is to [manually customize the OpenAPI output](https://postgrest.org/en/latest/references/api/openapi.html?highlight=openapi#overriding-full-openapi-response) and add the `"example"` key to each `"parameter"`, e.g:

```json
  "parameters": {
    ...
    "rowFilter.your_table.column": {
      "name": "column",
      "required": false,
      "format": "text",
      "in": "query",
      "type": "string",
      "example": "eq.Value"
    },
    ...
  }
```

We're not doing big changes to the OpenAPI output before #1698 is completed. I'll open an issue to keep track of this in the meantime.

**Reply by yevon** [reply]
Thanks! I will try this

**Comment by yevon** [comment]
I don't really understand how to override parts of the standard swagger resultset, the example in the docs states how to fully replace the response with a static json following openapi. Maybe I will wait until #1698 is complete. Thanks!
